lib/cli.js

```diff
@@ -21,16 +21,17 @@ class Cli {
   /**
    * Run the specified command
    * @param {String} cmd - the command to run
+   * @param {String} options - the command line options
    * @returns {Promise} a promise resolved when command finishes, or rejected with failure
    */
-  run (cmd) {
+  run (cmd, options) {
     const config = utils.getConfig()
     const vcs = this._getVcs(config)
     const ci = this._getCi(config, vcs)
     const bumper = this._getBumper({ci, config, vcs})
 
     if (cmd === 'bump') {
-      return bumper.bump()
+      return bumper.bump(options)
```

Comment:
I don't believe the `bump()` method has the options argument since your cleanup meaning you can completely remove the argument from here as well as in the `bin/cli.js` file.